supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -11,37 +11,38 @@
 
 
 [discrete]
-[[ibm-eserver-system-p]]
```

Comment:
Change required to the index `ISeries (noun)`?

![Screenshot from 2023-07-12 12-50-55](https://github.com/redhat-documentation/supplementary-style-guide/assets/57954076/3274d2b4-0641-4bb7-b1a1-be17deb43476)

Reply:
And:

![Screenshot from 2023-07-12 12-51-35](https://github.com/redhat-documentation/supplementary-style-guide/assets/57954076/e6e672ea-6501-4456-a69b-54cbf78f8ee2)

Reply:
I see the iSeries entry has been removed. The pSeries needs removal also?

Reply:
@lsmcfadden @manokuma can one of you please confirm? I would say yes it needs to be removed.

Reply:
Do you mean that you'll change it to  ~ _IBM Power_ represents the family of servers based on Power processors that can include combinations of IBM Power servers and its products which I mention above or 'remove' it altogether?

Reply:
@lsmcfadden I added the IBM Power entry under i in the RH SSG glossary replacing the IBM eserver system p entry [here](https://redhat-documentation.github.io/supplementary-style-guide/#_i). 
Therefore I would remove the pSeries entry entirely unless you object. (rebranding was back in 2008 and I haven't come across any entries in RH docs)

Reply:
I see that pSeries has been removed. Thank you.